settings.php

```diff
@@ -34,8 +34,8 @@
  * $config_directories['staging'] = '/home/myusername/config/staging';
  * @endcode
  */
-$config_directories['active'] = 'files/config_' . md5($database) . '/active';
-$config_directories['staging'] = 'files/config_' . md5($database) . '/staging';
+$config_directories['active'] = 'files/config_bed56320d59092fe62f20124c9583997/active';
+$config_directories['staging'] = 'files/config_bed56320d59092fe62f20124c9583997/staging';
```

Comment:
LOL. Exactly what we're trying to avoid with this fix. :wink:

Needs to be removed.